Bio/SeqIO/InsdcIO.py

```diff
@@ -1117,6 +1118,17 @@ def write_record(self, record):
         handle.write("FEATURES             Location/Qualifiers\n")
         rec_length = len(record)
         for feature in record.features:
+            # Handle any stray line breaks in the feature
+            for k in feature.qualifiers.keys():
+                if (
+                    isinstance(feature.qualifiers[k], Iterable)
```

Comment:
this line was added due to some tests failing when `qualifiers` was not an `Iterable` object (I think it was an int in one of the specs) so I added this extra guard.